server/edusync/src/main/java/com/codestates/edusync/model/study/studygroup/entity/Studygroup.java

```diff
@@ -68,7 +68,7 @@ public class Studygroup extends Auditable {
     @JoinColumn(name = "leader_member_id")
     private Member leaderMember;
 
-    @OneToMany(mappedBy = "studygroup", fetch = LAZY)
+    @OneToMany(mappedBy = "studygroup", cascade = {PERSIST, MERGE}, fetch = LAZY)
```

Comment:
```suggestion
    @OneToMany(mappedBy = "studygroup", cascade = {PERSIST, MERGE, REMOVE}, fetch = LAZY)
```
remove 도 있어야 스터디 그룹 삭제 될 때
join 된 것도 같이 삭제 되도록 해야 할 것 같습니다!